README.md

```diff
@@ -1 +1,7 @@
-# Hairven
+Hairven is an app for users to view collections of hairstyles and book appointments with saloons. Sounds boring. Let's complete this app.
```

Comment:
You'll need a section explaining how to install the application e.g `clone the repo`, `npm install` e.t.c.